docs/source/benchmarking/materials_science_and_engineering/vimmp.rst

```diff
@@ -1,5 +1,5 @@
 Virtual Materials Marketplace (VIMMP) Ontologies
```

Comment:
convert `Virtual Materials Marketplace (VIMMP) Ontologies` to `Virtual Materials Marketplace Ontology (VIMMP)`